File: INICIANTE/1020/main.dart
import 'dart:io';

void printFormatted(int days) {
  int years = days ~/ 365;
  days = days % 365;
  int months = days ~/ 30;
  days = days % 30;
  print("$years ano(s)");
  print("$months mes(es)");
  print("$days dia(s)");
}

void main() {
  int days = int.parse(stdin.readLineSync()!);
  printFormatted(days);
}

File: INICIANTE/1020/main.py
def print_formatted(days):
    years = days // 365
    days = days % 365
    months = days // 30
    days = days % 30
    print(f'{years} ano(s)')
    print(f'{months} mes(es)')
    print(f'{days} dia(s)')


days = int(input())
print_formatted(days)
